### DISCUSSION TITLE
Managing Two Instances of Mirth Connect on the Same Machine: Stopping Service of One Instance Affects the Other

### DISCUSSION THREAD

**Discussion by IvanPopov83** [nextgenhealthcare/connect#6219] [Q&A] [answered]
I have two instances of Mirth Connect installed on the same machine. When I run the Mirth Manager of the second instance (v.4.4.2) and try to stop the service, it stops the service of the first instance (v.3.6.2) instead. Why is this happening?

![image](https://github.com/nextgenhealthcare/connect/assets/171793702/fe9f8fb3-1eb7-4109-b1c2-f0c8671a6e39)

![image](https://github.com/nextgenhealthcare/connect/assets/171793702/33955e65-7422-4f06-8957-f7545359b627)

![image](https://github.com/nextgenhealthcare/connect/assets/171793702/b59e0b1f-8c0c-4472-b09c-2c19db72ae77)
![image](https://github.com/nextgenhealthcare/connect/assets/171793702/e9f4ef90-edc6-4d0d-a42f-35b7b644cbdc)
![image](https://github.com/nextgenhealthcare/connect/assets/171793702/c1e22992-0cfc-435b-82d0-b618df580fd3)

**Comment by ab-mg-23** [comment, marked as answer]
The MC Server Manager has the service name hardcoded to `Mirth Connect Service`. The MC Server Manager will always operate on `Mirth Connect Service` when issuing service commands.
